src/app/shared/object-list/item-list-element/item-list-element.component.ts

```diff
@@ -0,0 +1,22 @@
+import { Component } from '@angular/core';
+
+import { Item } from '../../../core/shared/item.model';
+import { renderElementsFor } from '../../object-collection/shared/dso-element-decorator';
+import { AbstractListableElementComponent } from '../../object-collection/shared/object-collection-element/abstract-listable-element.component';
+import { SetViewMode } from '../../view-mode';
+import { VIEW_MODE_ELEMENT } from '../../../+item-page/simple/related-items/related-items-component';
+
+@Component({
+  selector: 'ds-item-list-element',
+  styleUrls: ['./item-list-element.component.scss'],
+  templateUrl: './item-list-element.component.html'
+})
+
+/**
+ * The component used to list items depending on type
+ * Uses item-type-switcher to determine which components to use for displaying the list
+ */
+@renderElementsFor(Item, SetViewMode.List)
+export class ItemListElementComponent extends AbstractListableElementComponent<Item> {
```

Comment:
ItemListElementComponent => ItemTypeListElementComponent
I have this suggestion because you changed from Entity to ItemType, so instead of
calling EntityListElementComponent I think it should be called as ItemTypeListElementComponent

Reply:
> I have this suggestion because you changed from Entity to ItemType

I replaced "Entity" with "Item", so "EntityType" became "ItemType". 

As this component renders all Items when they're shown as an element of a list, I think the name `ItemListElementComponent` is appropriate.

Reply:
👍